worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/PlatformManager.java

```diff
@@ -298,10 +298,26 @@ public LocalConfiguration getConfiguration() {
         return queryCapability(Capability.CONFIGURATION).getConfiguration();
     }
 
+    /**
+     * Get the current supported {@link SideEffect}s.
+     *
+     * <p>If no platform has been registered yet, then a fallback value
+     * will be returned.</p>
```

Comment:
Isn't this wrong? Won't this _throw_ if no platforms are registered?

Reply:
yes, that was wrong. I fixed the wrong javadoc comment